source/manuals/grafana-dashboards.html.md.erb

```diff
@@ -1,58 +1,61 @@
 ---
-title: Viewing your metrics using Grafana dashboards
+title: View and create Grafana dashboards
 ---
 
 # <%= current_page.data.title %>
 
+*there should be a brief intro here - what is this content about, who’s it for?*
 
-## Logging into Grafana
+## Access Grafana
 
-Visit [grafana-paas.cloudapps.digital](https://grafana-paas.cloudapps.digital) and sign in with Google.
+Sign in to Grafana at [grafana-paas.cloudapps.digital](https://grafana-paas.cloudapps.digital) using your GDS Google account.
 
 ![google-auth](images/google-auth.png)
 
+## Display dashboards
 
-## Viewing your dashboards
+Team dashboards are organised into separate Grafana folders.
 
-Dashboards for teams have been organised into different Grafana folders.
+Select **Home** in the top left of the Grafana screen to choose your team’s dashboard folder and view your dashboards.
 
-Click 'Home' in the top left of the screen to pick your team's dashboard folder and see your dashboards.
+### Display using your team TV
 
+You should use your TV’s Google Chromebit user to display your Grafana dashboard. The Google Chromebit user only has read access to Grafana.
 
-## Creating and editing dashboards
+You should not use your personal Google account to display your Grafana dashboard on your team’s TV. Your personal Google account may have editing or admin permissions.
 
-In order to create or edit a dashboard in your team folder you will need to have the relevant permissions. Admin/editor permissions for your team folder can be granted by contacting us on the `#re-prometheus-support` Slack channel.
+## Create and edit dashboards
 
-If you don't have a team folder, contact us on the `#re-prometheus-support` Slack channel to create the folder and get admin rights to manage it. If you would like your own copy of any of the default dashboards that you can personalise then please ask for this too.
+Contact Reliability Engineering using the [#re-prometheus-support Slack channel][https://gds.slack.com/messages/CAF5H4N4Q/#] to request:
 
+* admin permissions to create or edit a dashboard in your team folder
+* a new team folder and the admin rights to manage it
+* copies of default dashboards to customise
 
 ## Default and example dashboards
 
-In the 'General' dashboard folder, we provide default and template dashboards for you to use. These are a good starting point for customisation or if you have just set up monitoring to check that everything works as expected.
+You can use the default and template dashboards in the **General** dashboard folder. These are a good starting point for customisation or if you have set up monitoring to check everything works as expected.
```

Comment:
What's the difference between default and template dashboards, we could say what they are.

Reply:
@soupdragon99 
Good question.

We currently have four dashboards in the General folder
- GDS Application Metrics Default Dashboard
- GDS Container Metrics Default Dashboard
- Postgres
- MySQL

The first two are dashboards that we originally thought teams would use for displaying their metrics but it seems to have diverged to them now creating their own dashboard in a team folder that may be based on the default dashboards. (I don't know if people still do use the Default dashboard as their only source of viewing metrics for their apps). They were made by our team to show what we think is best practice for viewing metrics from your applications or containers.

The Postgres and MySQL dashboards I believe are downloaded from the community dashboards and could be used by a team if they started to export Postgres or MySQL metrics. I don't think anyone is using them at the moment but we haven't deleted them.

I hope that gives you some context about why I have been thinking of the first two dashboards as 'default' and the second two as 'examples'. Saying that, we could just call them all 'default' or all 'example' dashboards to simplify things. I think 'example' might be a more descriptive word so would suggest that. Any thoughts?